.github/workflows/style.yml

```diff
@@ -48,3 +48,22 @@ jobs:
           filter_mode: diff_context
           fail_on_error: true
           exclude: "./.git/*" # Optional.
+
+  pylint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pylint
+
+      - name: Analysing the code with pylint
+        run: |
+          make pylint
```

Comment:
thank you. 
you can add a prop to this step to allow the job pass even on error, and comment something that should fix later, like:
```
# FIXME: fix the pylint errors in project and rm continue-on-error flag
continue-on-error: true
```

Reply:
Done